### PR TITLE
a31-add-weather

### PR DESCRIPTION
**Pull request by ghost**
creation of weather page

### CODE REVIEW

**Comment by Iregui44** [comment]
Why is this PR in draft?

File: src/jm/html-css/index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>A31-Weather</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+    />
+  </head>
+  <body>
+    <div class="page-blue">
```
Comment (Iregui44) [inline src/jm/html-css/index.html:14]: You are using div for everything, try and use more semantic html with tags, header, footer etc...

File: src/jm/html-css/page2.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
+body {
```
Comment (Iregui44) [inline src/jm/html-css/page2.css:1]: mmm I think is better to use a single css file for all styles in this case, but it is only a opinion.

File: src/jm/html-css/index.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      <div class="head">
+        <i class="fa-solid fa-dice-four"></i>
+        <i class="fa-solid fa-location-dot"> Minsk</i>
+        <i class="fa-solid fa-ellipsis-vertical"></i>
+      </div>
+      <button>Updating</button><br />
```
Comment (Iregui44) [inline src/jm/html-css/index.html:20]: dont forget aria-labels and href in all the tags that neeed it.

**Comment by Iregui44** [comment]
I like it a lot, well done.

File: src/jm/html-css/index.html (hunk 1 above)
Comment (Juanmediina) [reply src/jm/html-css/index.html]: more tags have been used

**Review by Iregui44 — APPROVED** [review]

**Pull request closed without merging**